resources/schema.py

```diff
@@ -265,6 +265,7 @@ class Meta:
     max_width = graphene.Float()
     max_length = graphene.Float()
     max_depth = graphene.Float()
+    number_of_places = graphene.Int()
```

Comment:
this will be `required=True` too, since the `Count` annotation will at least return `0`